### DISCUSSION TITLE
How to run auto segmentation without smoothing in 3D-Slicer based on MONAILabel

### DISCUSSION THREAD

**Discussion by ulphypro** [Project-MONAI/MONAILabel#1481] [Q&A]
Dear, all members!

I'm running auto segmentation using 3D-Slicer based on MONAILabel.

But when I clicked 'run' button in 'Auto Segmentation' option, it shows result that tumor softly handled as following attached image file.
![tumor_smoothing](https://github.com/Project-MONAI/MONAILabel/assets/67679322/7d71f915-dbe7-4e4f-bf7d-9205e4c02c0b)

I want to show rough tumor result without soft effect when I clicked 'run' button in Auto Segmentation option.

Is there the solution regarding problem above?

**Comment by tangy5** [comment]
This can be achieved with 3D Slicer UI, specifically: You could turn off the smoothing using the Set surface smoothing option, it's under the arrow next to the Show 3D button in Segment Editor.
![Screenshot from 2023-06-28 21-11-22](https://github.com/Project-MONAI/MONAILabel/assets/58751975/c7ac36f8-d859-4af6-96fe-543272be730b)

**Reply by ulphypro** [reply]
Dear @tangy5 

Thank you for answering my question.

It's delighted to solve the problem thanks to you.

**Comment by lassoan** [comment]
> I want show rough tumor result without soft effect

Note that we provide the option to skip surface smoothing to allow debugging or temporarily making the processing faster. However, smoothing is required to reconstruct the real shape of the object. The discrete samples that you can see if you skip smoothing is not the correct shape of the object.

Why do you want to show the incorrectly/incompletely reconstructed surface?

**Reply by ulphypro** [reply]
Dear @lassoan


Thank you for advising.

My chief ordered that he had wanted to see the tumor result into original rough shape.

**Reply by lassoan** [reply]
Thanks for the answer. It seems that you might think of the surface smoothing operation as some kind of optional post-processing step. It is actually a required step for reconstructing the correct shape from discrete samples. 

During digital imaging the signal is discretized but if the sampling was sufficiently dense then you can perfectly reconstruct the original continuous signal (the original surface) from the samples (voxels) using low-pass filtering (surface smoothing).

**Reply by ulphypro** [reply]
Thank you for detailed describing.
